src/main/java/com/programmers/springbootbasic/domain/voucher/domain/VoucherType/FixedAmountVoucher.java

```diff
@@ -0,0 +1,24 @@
+package com.programmers.springbootbasic.domain.voucher.domain.VoucherType;
+
+import static com.programmers.springbootbasic.exception.ErrorCode.INVALID_FIXED_VOUCHER_BENEFIT;
+
+import com.programmers.springbootbasic.exception.exceptionClass.VoucherException;
+
+public class FixedAmountVoucher implements VoucherType{
+
+    public FixedAmountVoucher(Integer benefit) {
+        if (benefit < 0) {
+            throw new VoucherException(INVALID_FIXED_VOUCHER_BENEFIT);
+        }
+    }
+
+    @Override
+    public String getVoucherTypeName() {
+        return "FIXED";
+    }
```

Comment:
```suggestion
    public VoucherTypeEnum getVoucherTypeName() {
        return FIXED;
    }
```

이미 Type을 enum으로 정의해둔 것으로 보이니 활용하여 유지보수성을 높이는 것이 좋을 듯합니다.

Reply:
[ca62e69](https://github.com/prgrms-be-devcourse/springboot-basic/pull/902/commits/ca62e69d5626e887b7d7bf2897c8a7b2cea7de74)

순환참조 문제때문에 이렇게 해놓고, 순환 참조 문제를 고친 이후에 변경하지 못했습니다! 현재는 변경 하였습니다!